src/sparkx/Particle.py

```diff
@@ -1076,6 +1078,27 @@ def compute_charge_from_pdg(self):
             return np.nan
         return PDGID(self.pdg).charge
 
+    def mT(self):
+        """
+        Compute the transverse mass :math:`m_{T}=\\sqrt{E^2-p_z^2}` of the particle.
+
+        Returns
+        -------
+        float
+            transverse mass
+
+        Notes
+        -----
+        If one of the needed particle quantities is not given, then `np.nan`
+        is returned.
+        """
+        if np.isnan(self.E) or np.isnan(self.pz):
+            return np.nan
+        elif abs(self.E) >= abs(self.pz):
+            return np.sqrt(self.E**2. - self.pz**2.)
+        else:
+            raise ValueError("|E| >= |pz| not fulfilled")
```

Comment:
Do we want to be so strict and raise a ValueError here? This would crash the analysis if it happens. We could do something similar to `compute_mass_from_energy_momentum` here.

Reply:
I think that we need a warning at least. If a user runs a script and gets numerical values without any signal that there is a severe issue, the user will never have a chance to understand its origin. Especially if the sparkx result is used for further analysis. 
I have also discussed this with @NGoetz who is fine with both, a warning or an error. But then I would also add it to `compute_mass_from_energy_momentum`.
What do you think?

Reply:
I agree that the user should be informed, but maybe with a warning instead of stopping the analysis. You can check if such a warning would be a good thing to implement in other `Particle` functions as well.

Reply:
Great, I will change it to a warning and check the other functions as well.

Reply:
@Hendrik1704 Which return value do you think makes sense and is easy identifiable for a user? What you still stay with 0. or something like a negative value or even nan?

Reply:
If we use nan, this particle would be filtered out by the filter functions automatically I think.